docs/notes/v6.0.0.md

```diff
@@ -0,0 +1,217 @@
+# MinIO Operator v6.0.0 release notes
+
+## About this release
+
+MinIO Operator is the recommended production-ready way to manage MinIO deployments in Kubernetes. This version includes many new features, bug fixes, new Kubernetes base version requirements, deprecations, and more.
+
+## What’s new?
+
+Operator 6.0.0 is now available. This release requires Kubernetes version 1.25.0 or later. If your cluster does not use Kubernetes version 1.25.0 or later, you must upgrade your cluster to before installing or upgrading to Operator v6.0.0+.
+
+* Introducing MinIO Job
+* Sidecar container
+* TLS refactors
+* Operator STS
+* Fields immutable
```

Comment:
This might make more sense to somebody who doesn't read our Operator repo issues all the time 😄 
```suggestion
* `servers` and `volumesPerServer` fields are immutable
```